core/src/test/java/org/apache/iceberg/rest/TestRESTViewCatalog.java

```diff
@@ -144,6 +151,39 @@ public void closeCatalog() throws Exception {
     }
   }
 
+  @Test
+  public void testPaginationForListViews() {
+    RESTCatalogAdapter adapter = Mockito.spy(new RESTCatalogAdapter(backendCatalog));
+    RESTCatalog catalog =
+        new RESTCatalog(SessionCatalog.SessionContext.createEmpty(), (config) -> adapter);
+    catalog.initialize("test", ImmutableMap.of(RESTSessionCatalog.REST_PAGE_SIZE, "10"));
+
+    int numberOfItems = 100;
+    String namespaceName = "newdb";
+    String tableName = "newtable";
+    String viewName = "newview";
+
+    TableIdentifier tableIdentifier = TableIdentifier.of(namespaceName, tableName);
+
+    // create initial namespace and table
+    catalog().createNamespace(tableIdentifier.namespace());
+    tableCatalog().buildTable(tableIdentifier, SCHEMA).create();
```

Comment:
we don't need to create the table. The query of the view can just have a random SQL

Reply:
will fix